src/librustc_trans/save/dump_csv.rs

```diff
@@ -653,8 +647,8 @@ impl <'l, 'tcx> DumpCsvVisitor<'l, 'tcx> {
         }
     }
 
-    fn process_mod(&mut self,
-                   item: &ast::Item) {  // The module in question, represented as an item.
+    // `item` is the module in question, represented as an item.
```

Comment:
@nrc did you notice that it moved the comment from inside to outside of the function? Is this intentional?